Civi/Api4/Utils/CoreUtil.php

```diff
@@ -390,14 +390,7 @@ public static function getEntityTypes(): array {
    * @return array
    */
   public static function getOptionValueFields($optionGroup, $key = 'name'): array {
-    // Prevent crash during upgrade
-    if (array_key_exists('option_value_fields', \CRM_Core_DAO_OptionGroup::getSupportedFields())) {
-      $fields = \CRM_Core_DAO::getFieldValue('CRM_Core_DAO_OptionGroup', $optionGroup, 'option_value_fields', $key);
-    }
-    if (!isset($fields)) {
-      return ['name', 'label', 'description'];
-    }
-    return explode(',', $fields);
+    return \CRM_Core_DAO_OptionGroup::getDbVal('option_value_fields', $optionGroup, $key) ?: ['name', 'label', 'description'];
```

Comment:
@ufundo this isn't really part of the fix, just something I noticed: these 7 sloc can be eliminated by calling this new `getDbVal` function which does the same thing.

Can do a separate PR for this if desired.